core/src/main/java/org/stellar/anchor/sep10/Sep10Service.java

```diff
@@ -65,6 +65,12 @@ public ChallengeResponse createChallenge(ChallengeRequest challengeRequest) thro
           sep10Config.getOmnibusAccountList().contains(challengeRequest.getAccount().trim());
     }
 
+    if (sep10Config.isRequireKnownOmnibusAccount() && !omnibusWallet){
+      // validate that requesting account is allowed access
+      infoF("requesting account: {} is not in allow list", challengeRequest.getAccount().trim());
```

Comment:
🤔 Since we're returning an error here, what about logging with `errorF` instead of `infoF`? This could make it easier to debug when the authentication fails?

Reply:
i think we should stick with infoF here since the `error` thrown isn't is system error but rather invalid user input. this also messes with metrics and makes it seem like theres an issue with the system rather than user error